config/locales/pwpx.en.yml

```diff
@@ -136,6 +136,14 @@ en:
       save: Save Policy
       team_policy: Team Policy
       team_policy_help: A team policy is a set of rules that apply to all pushes made by members of this team.
+  activerecord:
```

Comment:
These parts are not necessary to use `Model.human_attribute_name(attribute_name)` correctly because `:name` attribute can be transformed into a readable format(`name`) for English. But, I'm not sure how it behaves for different languages. So, I added this translations explicitly.